apps/journeys-admin/pages/journeys/[journeySlug].tsx

```diff
@@ -33,6 +37,15 @@ function SingleJourneyPage({ journey }: SingleJourneyPageProps): ReactElement {
         <Typography variant={'h6'}>{journey.status}</Typography>
         <Typography variant={'h6'}>created: {journey.createdAt}</Typography>
         <Typography variant={'h6'}>published: {journey.publishedAt}</Typography>
+
+        <CardOverview
+          slug={journey.slug}
+          blocks={
```

Comment:
```suggestion
          blocks={blocks as Array<TreeBlock<StepBlock>>}
```
and on line 22:

const blocks = journey.blocks !== null ? transformer(journey.blocks) : []